src/index.css

```diff
@@ -322,6 +322,7 @@ small {
 
 .search-wrapper {
   margin-top: 35px !important;
+  margin-bottom: 20px !important;
```

Comment:
You don't have to make any changes for this comment, the downstream impact is pretty trivial, but it's a great learning example. Open up a tab with your branch, and then another side by side on dev and view the admin manage DAC page. This css change slightly pushes down the table below the search field. This is the primary reason I push for moving our styles to inline styles. If you make a change to the inline style on a component, you won't impact any other pages/components in any way, either positive or negative. If you make a change to a css file, you need to visually assess the impact in all pages where that style class is used. In this case, the search wrapper is used quite a bit.

Reply:
The second big reason I push for inline styles are that they are tremendously flexible. I can refactor them easily, find all usages easily, merge a base style with new properties easily, filter out/replace existing properties, etc. just using standard javascript. The only real limitation to this approach is css state. We need to manage things like `hover` state via javascript instead of css.